Azure.ResourceManager.Core/ResourceOperationsBase.cs

```diff
@@ -133,5 +134,10 @@ protected void UpdateTags(string key, string value, IDictionary<string, string>
                 existingTags.Add(key, value);
             }
         }
+
+        /// <summary>
+        /// Gets the resource client.
+        /// </summary>
+        protected ResourcesManagementClient ResourcesClient => new ResourcesManagementClient(BaseUri, Id.Subscription, Credential);
```

Comment:
if we move this to operationsbase.cs we won't have to duplicated it in locationcontainer or subscriptioncontainer